### ISSUE TITLE
Sync complete method and/or callback

### ISSUE THREAD

**Issue by mattheworiordan** [ably/ably-java#20]
*\* LOW PRIORITY ALERT **

In the Java library, how can a developer:
- Check that the presence map sync is complete?
- Subscribe to a callback once the sync is complete?

In Ruby, there is:
- [Presence#sync_complete? predicate method](http://www.rubydoc.info/gems/ably/0.8.1/Ably/Realtime/Presence#sync_complete%3F-instance_method)
- [Presence#members.once(:in_sync)](http://www.rubydoc.info/gems/ably/0.8.1/Ably/Realtime/Presence/MembersMap) this is undocumented as the MembersMap is a state machine of sorts and will reach an `in_sync` state.
- Once can also do [Presence#get(callback)](http://www.rubydoc.info/gems/ably/0.8.1/Ably/Realtime/Presence#get-instance_method) which by default waits until a sync is complete before calling the callback with an array of members

Do you think we should formalise two methods:
- Presence#inSync or perhaps Presence#syncComplete - boolean indicating whether the sync has completed
- Presence#onSync(callback)

**Comment by mattheworiordan** [comment]
This is part of the spec http://docs.ably.io/client-lib-development-guide/features/#RTP13

**Comment by paddybyers** [comment]
This is implemented; closing
